pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1600,27 +1605,17 @@ func isValidCIDRRange(CIDR *net.IPNet) bool {
 // checks carried out :
 // 		* has a cidr range between \16 and \26
 // 		* does not overlap with cluster vpc cidr block
-func validateStandaloneCidrBlock(ctx context.Context, client client.Client, ec2Api ec2iface.EC2API, logger *logrus.Entry, vpcCidrBlock *net.IPNet) error {
+func validateStandaloneCidrBlock(validateCIDR *net.IPNet, clusterCIDR *net.IPNet) error {
 	// validate has a cidr range between \16 and \26
-	if !isValidCIDRRange(vpcCidrBlock) {
-		return errorUtil.New(fmt.Sprintf("%s is out of range, block sizes must be between `/16` and `/26`, please update `_network` strategy", vpcCidrBlock.String()))
-	}
-
-	clusterVPC, err := getClusterVpc(ctx, client, ec2Api, logger)
```

Comment:
I'm moving this out because we now have a loop where this function could be called more than once.